src/services/apiService.js

```diff
@@ -71,3 +72,31 @@ export const startQuizSession = (quizId) =>
       toast.error('An error occurred while starting the quiz');
       console.error(`Error starting quiz: ${e}`);
     });
+
+export const fetchRecordings = () =>
+  axios
+    .get(`https://api.edukona.com/recordings/`, {
+      headers: {
+        Authorization: `Token ${token}`,
+      },
+    })
+    .then((res) => res.data.recordings)
+    .catch((error) => {
+      console.error(error);
+      return [];
+    });
+
+export const deleteRecording = (recording) => {
+  axios
+    .delete(`https://api.edukona.com/recordings/${recording}/delete-recording`, {
+      headers: {
+        Authorization: `Token ${token}`,
+      },
+    })
+    .then((res) =>
+      res.status === 200
+        ? toast.success('Recording successfully deleted!', { icon: '🗑️', theme })
+        : toast.error('Could not delete recording.', { theme })
+    )
+    .catch((error) => console.error(error));
+};
```

Comment:
Similar to my comment in the other PR, this should be consistent with the other functions `api.get/delete`; that'll also make it easier for us to swap the base URL with the pre-prod once we get to that point.